### ISSUE TITLE
Subcommunities listing

### ISSUE THREAD

**Issue by elletjies** [carpentries/carpentries.org#49]
**Header**: The Carpentries Subcommunities

Please add all the info on this page - https://carpentries.org/community-groups/ (I am not copying it here since it is not that easy as copy and paste for the registry)

**Header**: Regional Communities Mailing Lists: (https://docs.carpentries.org/topic_folders/regional_communities/regional_email_lists.html)

## Regional Mailing Lists

All mailing lists for [The Carpentries](https://carpentries.topicbox.com/groups) are hosted on Topicbox.  This includes a number of mailing lists specific to local or regional communities.

#### Africa
- [Africa](https://carpentries.topicbox.com/groups/local-africa)
- [Middle-East and North Africa](https://carpentries.topicbox.com/groups/local-middle-east)

#### Asia
- [Bangladesh](https://carpentries.topicbox.com/groups/local-bangladesh)
- [India](https://carpentries.topicbox.com/groups/local-india)

#### Australia/New Zealand
- [Australia and New Zealand](https://carpentries.topicbox.com/groups/local-aunz)

#### Canada
- [Vancouver](https://carpentries.topicbox.com/groups/local-vancouver)

#### Europe
- [Germany](https://carpentries.topicbox.com/groups/local-germany)
- [Heidelberg](https://carpentries.topicbox.com/groups/local-heidelberg)
- [United Kingdom](https://carpentries.topicbox.com/groups/local-uk)
- [Nordic countries](https://carpentries.topicbox.com/groups/local-nordic)

#### Latin America
- [Latin America](https://carpentries.topicbox.com/groups/local-latinoamerica)
- [Puerto Rico](https://carpentries.topicbox.com/groups/local-puertorico)

#### United States
- [Bay Area](https://carpentries.topicbox.com/groups/local-bayarea)
- [Boston](https://carpentries.topicbox.com/groups/local-boston)
- [Colorado](https://carpentries.topicbox.com/groups/local-colorado)
- [Davis, CA](https://carpentries.topicbox.com/groups/local-davis)
- [Lansing, MI](https://carpentries.topicbox.com/groups/local-lansing)
- [Merced, CA](https://carpentries.topicbox.com/groups/local-merced)
- [Seattle, WA](https://carpentries.topicbox.com/groups/local-seattle)
- [Southeast US Library Carpentry](https://carpentries.topicbox.com/groups/local-libcarpentry-southeast-u)
- [Southern California](https://carpentries.topicbox.com/groups/local-socal)
- [Southwest United States](https://carpentries.topicbox.com/groups/local-swusa)
- [University of California](https://carpentries.topicbox.com/groups/local-uc)
- [Washington DC Area](https://carpentries.topicbox.com/groups/local-dc) 

Can these two headers also go in the menu on the right (not sure if this is automated in the template)

**Comment by maneesha** [comment]
@elletjies Please see the page here: https://carp-new-website.netlify.app/community/get-connected/

For the subcommunities, I am still working on some formatting (like how you currently see "Contact" or "Connect" even if they don't have any information there.

Please share any other feedback here.

**Comment by elletjies** [comment]
I had a look and the info is good to go. It is a lot so was wondering if we could collapse the subcommunity descriptions so that when you click on an arrow or something it pops up. Also, would it be possible to add a country flag where appropriate (for the subcommunities as well as regional community content?)? The info does not look visually pleasing to the eye.

**Comment by elletjies** [comment]
Please let me know if this is out of scope for this round of reviews

**Comment by maneesha** [comment]
@elletjies Yes, this is out of scope for the current review.  I think you good ideas though, so I am keeping this issue open to come back to.

**Comment by maneesha** [comment]
@OscarSiba will bring this up with CET.

**Comment by maneesha** [comment]
Discussed with @OscarSiba. He will be working on setting up [subcommunities](https://carpentries.org/community/get-connected/) into three categories: 

- Regional - National (Geographical)
- Local - institutional level (Universities or research groups) 
- SIG (Special Focus)

Each category will have a heading and short description with accordions for each subcommunity.

**Comment by maneesha** [comment]
@OscarSiba please assign to me once you have the lists of which subcommunities should go in what category.  I can guess but you know them best so I want to be sure to get it right.

**Comment by OscarSiba** [comment]
Can we please classify the subcommunuties as below. 

The subcommunities that I have italicised (Heidelberg, Midwest (US), and US-Northeast) I would like your views @maneesha @elletjies as to whether they should be classified as Local or Regional Subcommunities.  Currently I have placed them under Local, but they may qualify to be regional, and I would then have to modify the category descriptions to reflect this re-classification. Please advise.  

**Subcommunities**

**Regional Subcommunities**

The subcommunities listed in this section are subcommunities whose focus is Carpentries activities at a specified large geographical - regional or national - area. 

* Carpentries Italia
* The Carpentries in Australia
* New Zealand/Aotearoa
* Carpentries-ja
* The Carpentries in Africa
* DACH-Region
* Carpentries México
* Carpentries-Israel

**Local Subcommunities**

The subcommunities listed in this section are subcommunities whose focus is Carpentries activities at a specified local and institutional level - a province, district, town or city within a country, or at a university or research institution. 

* Carpentries at Stanford University
* UW-Madison Carpentries Community
* Smithsonian Institution Carpentries
* *Heidelberg Carpentries*
* University of California Santa Barbara
* Carpentry @ UiO
* Carpentries @ VT
* *Midwest (US) Carpentries Community*
* *Carpentries US-Northeast*

**Special Interest Groups**

The subcommunities listed in this section are subcommunities whose focus is Carpentries activities around a specific area of interest to The Carpentries community, whether this activity is conducted at a local, regional or global level. 

* PyAOS Carpentry
* Carpentries Offline
* HPC Carpentry
* Carpentries Ukraine

**Comment by elletjies** [comment]
@OscarSiba 

- Midwest (US) Carpentries Community
- Carpentries US-Northeast

It might also be Regional. @maneesha what do you think?

**Comment by maneesha** [comment]
It's a little subjective but I would probably consider those to be regional.  I would think of "local" as something you could go to for the afternoon and "regional" is a day trip or longer.

As I'm looking at this and thinking about it more, do we want to introduce some consistency in how subcommunities named?  For example, instead of Midwest (US) Carpentries Community and Carpentries US-Northeast, maybe have them be _Carpentries US (Midwest)_ and _Carpentries US (Northeast)_.
Or for countries, should we consistently say _The Carpentries [country]_?

**Comment by OscarSiba** [comment]
I agree on a consistent naming conventions, but didn't want to change anything because I wasn't sure if that is how the subcommunities refer to themselves as, and therefore they have autonomy over that? It's the subcommunities that add their details, right @elletjies, and do you think we can change the names or will you need to first consult with them?

**Comment by elletjies** [comment]
@OscarSiba, I am not sure I understand what you mean by changing their names. How I understand what we are aiming to do is to categorise the subcommunities right? Col D of this document might help - https://docs.google.com/spreadsheets/d/1GxV0lRTK4EoLuYibf9RSZjETyQPrNUNeOP73kK2lLs4/edit?resourcekey=&gid=884349548#gid=884349548

**Comment by OscarSiba** [comment]
Hi @elletjies. I am referring to @maneesha's comment on "do we want to introduce some consistency in how subcommunities named? For example, instead of Midwest (US) Carpentries Community and Carpentries US-Northeast, maybe have them be Carpentries US (Midwest) and Carpentries US (Northeast).  Or for countries, should we consistently say The Carpentries [country]?"

And I am saying, if the subcommunity entered the name in the spreadsheet as it is reflected on the current website, can we change that name so that it reflects a standardised naming convention that we decide to adopt as part of this process we are doing now or will you need to first reach out to them to confirm that it is ok for us to rename the subcommunity in a standard way? 

The revised list:

**Subcommunities**

**Regional Subcommunities**

The subcommunities listed in this section are subcommunities whose focus is Carpentries activities at a specified large geographical - regional or national or provincial or district - area.

Carpentries Italia
The Carpentries in Australia
New Zealand/Aotearoa
Carpentries-ja
The Carpentries in Africa
DACH-Region
Carpentries México
Carpentries-Israel
Midwest (US) Carpentries Community
Carpentries US-Northeast

**Local Subcommunities**

The subcommunities listed in this section are subcommunities whose focus is Carpentries activities at a specified local and institutional level - town or city or at a university or research institution.

Carpentries at Stanford University
UW-Madison Carpentries Community
Smithsonian Institution Carpentries
Heidelberg Carpentries
University of California Santa Barbara
Carpentry @ UiO
Carpentries @ VT

**Special Interest Groups**

The subcommunities listed in this section are subcommunities whose focus is Carpentries activities around a specific area of interest to The Carpentries community, whether this activity is conducted at a local, regional or global level.

PyAOS Carpentry
Carpentries Offline
HPC Carpentry
Carpentries Ukraine

**Comment by elletjies** [comment]
@OscarSiba, thank you for clarifying. I think we should keep the naming convention as they entered the register. The amended list looks good!

**Comment by maneesha** [comment]
@OscarSiba  Carpentres Ukraine and Carpentries-ja are both listed as groups involved in lesson translations, so not necessarily local or regional.  You've listed Ukraine as special interest and Carpentries-ja as regional.  Is that supposed to be different?

**Comment by maneesha** [comment]
Also @OscarSiba  you have Carpentries-ja as regional but their description says they are around the world.

**Comment by elletjies** [comment]
@maneesha both identify to be connected by language (Carpentres Ukraine is based in the UK). If I am not mistaken, they are mostly involved in translation efforts.

**Comment by OscarSiba** [comment]
Thanks for picking this up @maneesha. Please move -ja to Special Interest Groups category.

**Comment by maneesha** [comment]
Addressed with #433
